pkg/cmd/openshift-controller-manager/cmd.go

```diff
@@ -131,5 +130,9 @@ func (o *OpenShiftControllerManager) RunControllerManager() error {
 	if err != nil {
 		return err
 	}
-	return RunOpenShiftControllerManager(config, clientConfig)
+	if err = RunOpenShiftControllerManager(config, clientConfig); err != nil {
+		return err
+	}
+	// TODO: Split RouteController into its own config and command.
+	return RunRouteControllerManager(config, clientConfig)
```

Comment:
instead of this, add `RunRouteControllerManager` to `ControllerInitializers` in `pkg/cmd/controller/config.go` and remove the lines for 

```go
	"openshift.io/ingress-ip":       RunIngressIPController,
	"openshift.io/ingress-to-route": RunIngressToRouteController,
```

This way, the main controller launcher will launch the logic that gets the lease and starts the route controllers.